test/PackageUtilities.js

```diff
@@ -4,134 +4,186 @@ import path from "path";
 import PackageUtilities from "../src/PackageUtilities";
 import Package from "../src/Package";
 import Repository from "../src/Repository";
-import initFixture from "./_initFixture";
+import initFixture from "./helpers/initFixture";
 
 describe("PackageUtilities", () => {
-  let testDir;
-
-  beforeEach((done) => {
-    testDir = initFixture("PackageUtilities/basic", done);
-  });
-
-  describe(".getPackagesPath()", () => {
-    it("should append the packages path to the repo path given", () => {
-      assert.equal(
-        PackageUtilities.getPackagesPath("/path/to/repo"),
-        path.join("/path/to/repo/packages")
-      );
+  describe("public API", () => {
+    let testDir;
+
+    beforeEach(() => initFixture("PackageUtilities/basic").then((dir) => {
```

Comment:
This fixture used to be initialized for _every_ test in this file, despite over half of them doing another fixture init (for filtering) and ignoring this. Seemed wasteful.